song_translator/track/urls.py

```diff
@@ -0,0 +1,15 @@
+from django.urls import path
+from rest_framework.urlpatterns import format_suffix_patterns
+
+from .views import TrackList, TrackDetail, SingerList, SingerDetail, TrackTranslate
+
+
+urlpatterns = [
+    path('tracks/', TrackList.as_view()),
+    path('tracks/<int:pk>/', TrackDetail.as_view()),
+    path('tracks/<int:pk>/<str:lang>', TrackTranslate.as_view()),
```

Comment:
let's restrict languages that could be used in urls for example like this. https://stackoverflow.com/a/6253248/3888551